renderers/main.py

```diff
@@ -214,16 +173,127 @@ def __draw_game(self):
+            return
+
+        if self.data.standings.is_postseason() and self.canvas.width <= 32:
+            return
+
+        update = 1
+        while cond():
+            if self.data.standings.is_postseason():
+                standings.render_bracket(
+                    self.canvas,
+                    self.data.config.layout,
+                    self.data.config.scoreboard_colors,
+                    self.data.standings.leagues[self.standings_league],
+                )
+            else:
+                standings.render_standings(
+                    self.canvas,
+                    self.data.config.layout,
+                    self.data.config.scoreboard_colors,
+                    self.data.standings.current_standings(),
+                    self.standings_stat,
+                )
+
+            if self.data.network_issues:
+                network.render_network_error(self.canvas, self.data.config.layout, self.data.config.scoreboard_colors)
+
+            self.canvas = self.matrix.SwapOnVSync(self.canvas)
+
+            if self.data.standings.is_postseason() and update % 20 == 0:
+                if self.standings_league == "NL":
+                    self.standings_league = "AL"
+                else:
+                    self.standings_league = "NL"
+            elif self.canvas.width == 32 and update % 5 == 0:
+                if self.standings_stat == "w":
+                    self.standings_stat = "l"
+                else:
+                    self.standings_stat = "w"
+                    self.data.standings.advance_to_next_standings()
+            elif self.canvas.width > 32 and update % 10 == 0:
+                self.data.standings.advance_to_next_standings()
+
+            time.sleep(1)
+            update = (update + 1) % 100
+
     def __max_scroll_x(self, scroll_coords):
         scroll_max_x = scroll_coords["x"] + scroll_coords["width"]
         self.scrolling_text_pos = min(scroll_max_x, self.scrolling_text_pos)
 
     def __update_scrolling_text_pos(self, new_pos, end):
-        """Updates the position of the probable starting pitcher text."""
+        """Updates the position of scrolling text"""
         pos_after_scroll = self.scrolling_text_pos - 1
         if pos_after_scroll + new_pos < 0:
             self.data.scrolling_finished = True
             if pos_after_scroll + new_pos < -10:
                 self.scrolling_text_pos = end
                 return
         self.scrolling_text_pos = pos_after_scroll
+
+    def no_games_cond(self) -> bool:
+        """A condition that is true only while there are no games live"""
+        return not self.data.schedule.games_live()
+
+
+def permanent_cond() -> bool:
+    """A condition that is always true"""
+    return True
+
+
+def timer_cond(seconds, refresh=1) -> Callable[[], bool]:
+    """Create a condition that is true for the specified number of seconds"""
+    curr = 0
+
+    def cond():
+        nonlocal curr
+        curr += refresh
```

Comment:
This doesn't seem accurate. Why not just check current timestamp - started timestamp and compare to target?

Reply:
This is exactly the kind of question I love code review for. The answer to "why not" is "I didn't think of that".

Way better.